roles/users/tasks/main.yml

```diff
@@ -43,5 +43,8 @@
     user: "{{ item.name }}"
     key: "{{ item.key }}"
   with_items: managed_users|list + managed_admin_users|list
+  # Register and retry to work around transient githubusercontent.com issues
+  register: ssh_key_update
+  until: ssh_key_update.state == 'present'
```

Comment:
I realize the default for ``until`` is 3 retries with a 5 second delay between, but maybe it'd be better to still explicitly define them here?  If nothing else it'd make this block easier to read for somebody who doesn't fully understand how ``until`` works.